src/client/backends/Msfs/Mapping.ts

```diff
@@ -294,13 +299,14 @@ export class MsfsMapping {
         return Array.from(ils.values()).filter((ils) => !!ils).map((ils) => {
             const icao = ils.icao.trim();
             return {
+                fixType: FixType.IlsNavaid,
```

Comment:
This is also not a fix so no changes needed to this type. VhfNavaids cover this need.